rebar.config

```diff
@@ -2,14 +2,28 @@
 {erl_opts, [debug_info]}.
 
 {deps, [
-    {rosie_rclerl, "*.", {git, "https://github.com/rosie-project/rosie_rclerl.git", {branch, "main"}}},
+    {rosie_dds, "*.", {git, "https://github.com/rosie-project/rosie_dds.git", {branch, "main"}}},
+    {rosie_rclerl, "*.",
+        {git, "https://github.com/rosie-project/rosie_rclerl.git", {branch, "main"}}},
     {example_interfaces, {ros2, galactic}},
     {std_msgs, {ros2, galactic}},
-    {geometry_msgs, {ros2, galactic}}
+    {geometry_msgs, {ros2, galactic}},
+    {rosie_demos_msgs, {ros2, local}}
```

Comment:
For most user who will clone compile this project, fectching `rosie_demos_msgs` will fail with a rebar3 api error, stopping the compilation. This `ros2, local` dep does not really work well in this rebar3 project.
It basically forces all umbrella apps to depend on a ros2 package and depend on a colcon setup...
My suggestion is to print a warning and let the compilation go on .... OR
you can add a profile to the config so that  `{rosie_demos_msgs, {ros2, local}}` is searched only if the user wants to compile/shell/release for that profile. 
Please test what happens if `rosie_demos_msgs`and/or ros2 are not installed.
The apps that do not need to use ros2 or ros2 local packages should still be usable.
If there is no clean solution for this we may consider to add a separate repo for these `rosie_demos apps` that depend on `colcon` and a `ros2` installation.

Reply:
I think the best solution is to delete the `counter_action_server` app because even putting it in profile the project doesn't compile because the `counter_action_server` dependencies are not resolved